docs/_includes/anchor-links.html

```diff
@@ -1,9 +1,9 @@
 <script>
   let anchorForId = function (id) {
     let anchor = document.createElement("a");
-    anchor.className = "header-link";
+    anchor.className = "anchor-link";
     anchor.href      = "#" + id;
-    anchor.innerHTML = "<span class=\"sr-only\"></span><img class=\"anchor-link\" src=\"/assets/images/anchor-link.svg\" alt=\"\"></img>";
+    anchor.innerHTML = "<span class=\"sr-only\"></span><img class=\"anchor-icon\" src=\"/assets/images/anchor-icon.svg\" alt=\"\"></img>";
```

Comment:
What does the `sr-only` class on the `span` do? I understand that it's for screen readers, but not sure of its specific purpose here. Is it supposed to hide the icon?

Reply:
```suggestion
    anchor.innerHTML = "<span class=\"sr-only\">Permalink</span><img class=\"anchor-icon\" src=\"/assets/images/anchor-icon.svg\" alt=\"\"></img>";
```

Good catch. I was missing screenreader text information.

Reply:
Oh, that explains everything!